src/game.rs

```diff
@@ -45,7 +45,29 @@ pub mod flags {
 ///
 /// [http://docs.screeps.com/api/#Game.rooms]: http://docs.screeps.com/api/#Game.rooms
 pub mod rooms {
-    game_map_access!(objects::Room, Game.rooms);
+    use std::collections::HashMap;
+
+    use crate::{local::RoomName, macros::*, objects::Room};
+
+    /// Retrieve the full `HashMap<RoomName, Room>`.
+    pub fn hashmap() -> HashMap<String, Room> {
+        js_unwrap!($js_inner)
```

Comment:
That looks like a leftover from the macro call?

Reply:
Right, thanks!